.ci/Jenkinsfile

```
@@ -231,7 +231,7 @@ pipeline {
   post {
     cleanup {
       githubCheckNotify(currentBuild.currentResult == 'SUCCESS' ? 'SUCCESS' : 'FAILURE')
-      notifyBuildResult(analyzeFlakey: true, flakyReportIdx: "reporter-e2e-tests-end-2-end-tests-pipeline-master", prComment: true)
+      notifyBuildResult(analyzeFlakey: true, flakyReportIdx: "reporter-e2e-tests-end-2-end-tests-pipeline-7.10.x", prComment: true)
```

What do you think to customise the index for branches or PRs, for branches we can use the BRANCH_NAME for PRs then we use the CHANGE_TARGET. (see https://beats-ci.elastic.co/pipeline-syntax/globals#env)

```suggestion
      whenTrue(isBranch()) {
         setEnvVar("SUFFIX", env.BRANCH_NAME))
      }
      whenTrue(isPR()) {
         setEnvVar("SUFFIX", env.CHANGE_TARGET))
      }
      notifyBuildResult(analyzeFlakey: (isBranch() || isPR()), flakyReportIdx: "reporter-e2e-tests-end-2-end-tests-pipeline-${SUFFIX}", prComment: true)
```

---

Would it be `${env.SUFFIX}`, or it doesn't matter?

---

@v1v I saw this old comment about the usage of the environment variable. Is it valid?

---

@v1v let's merge this PR as is, and create a _backportable_ PR with your suggestion. wdyt?